subsys/bluetooth/controller/ll_sw/ull_adv_aux.c

```diff
@@ -116,7 +116,9 @@ uint8_t ll_adv_aux_ad_data_set(uint8_t handle, uint8_t op, uint8_t frag_pref,
 	struct pdu_adv *pdu_prev;
 	uint8_t ad_len_overflow;
 	uint8_t ad_len_chain;
+	uint8_t chain_err = 0U;
 	struct pdu_adv *pdu;
+	uint8_t ad_len = 0U, ad_len_prev = 0U;
```

Comment:
May be these too can move closer to locality of reference? (to reduce program stack depth)

Reply:
only ad_len_prev can be moved